scipy/special/basic.py

```diff
@@ -754,20 +754,97 @@ def riccati_yn(n, x):
 
 
 def erfinv(y):
-    """Inverse function for erf.
+    """Inverse error function.
+    This function compute the inverse error function.
+    Inverse error function is satisfying :math:`erf(erf^{-1}(x))=x`.
+
+    Parameters
+    ----------
+    y : ndarray
+        Argument at which to evaluate for (-1, 1)
+
+    Returns
+    -------
+    erfinv : ndarray
+        Value of erfinv(y0), ..., erfinv(yn)
+
+    Examples
+    --------
+    1) evaluating a float number
+
+    >>> from scipy import special
+    >>> special.erfinv(0.5)
+    0.4769362762044698
+
+    2) evaluating a ndarray
+
+    >>> from scipy import special
+    >>> import numpy as np
+    >>> y = np.linspace(-1.0, 1.0, num=10)
+    >>> special.erfinv(y)
+    array([       -inf, -0.86312307, -0.5407314 , -0.30457019, -0.0987901 ,
+            0.0987901 ,  0.30457019,  0.5407314 ,  0.86312307,         inf])
+
     """
     return ndtri((y+1)/2.0)/sqrt(2)
 
 
 def erfcinv(y):
-    """Inverse function for erfc.
+    """Inverse complementary error function.
+    This function compute the inverse error function.
+    It is related to inverse error function by :math:`erfc^{-1}(1-x)=erf^{-1}(x)`
+
+    Parameters
+    ----------
+    y : ndarray
+        Argument at which to evaluate for (0, 2)
```

Comment:
What does `evaluate for (0, 2)` intended  to indicate here?  Is the domain the open interval `(0, 2)`?  Perhaps the closed interval `[0, 2]`?

Reply:
I intend `evaluate for (0, 2)` for the domain the open interval (0, 2).
Should I more clearly depict the expression?